nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -89,13 +92,29 @@ in {
         example = "45min";
         description = ''
           Add a randomized delay before each automatic upgrade.
-          The delay will be chozen between zero and this value.
+          The delay will be chosen between zero and this value.
           This value must be a time span in the format specified by
           <citerefentry><refentrytitle>systemd.time</refentrytitle>
           <manvolnum>7</manvolnum></citerefentry>
         '';
       };
 
+      persistent = mkOption {
+        default = true;
+        type = types.bool;
+        example = false;
+        description = ''
+          Takes a boolean argument. If true, the time when the service
+          unit was last triggered is stored on disk. When the timer is
+          activated, the service unit is triggered immediately if it
+          would have been triggered at least once during the time when
+          the timer was inactive. Such triggering is nonetheless
+          subject to the delay imposed by RandomizedDelaySec=. This is
+          useful to catch up on missed runs of the service when the
+          system was powered down.
+        '';
```

Comment:
Same here, think it needs documentation, but I think this is less controversial since `persistent = true` makes more sense for this service.

Reply:
Do you know what documentation (where) exactly? If I can, I would update it.

Reply:
https://www.freedesktop.org/software/systemd/man/systemd.timer.html#Persistent=

Reply:
I don't know exactly, I think maybe it is only filled after the release is tagged :thinking: ?

Reply:
Release notes for Okapi are over there:

https://github.com/NixOS/nixpkgs/blob/7b968c79a15796c524b75a0e43d643e36aa10e97/nixos/doc/manual/release-notes/rl-2103.xml#L86